invenio_app_ils/schemas/items/item-v1.0.0.json

```diff
@@ -46,52 +55,53 @@
         }
       }
     },
-    "legacy_id": {
-      "type": "string",
-      "title": "The previous id of this Item before migration"
-    },
-    "legacy_library_id": {
+    "pid": {
       "type": "string",
-      "title": "The previous id of the location of this Item before migration"
+      "title": "The persistent identifier of this Item"
     },
     "barcode": {
       "type": "string",
       "title": "The barcode of this Item"
     },
-    "shelf": {
-      "type": "string",
-      "title": "The shelf location of this Item"
-    },
     "description": {
       "type": "string",
-      "title": "Any extra description for this ItItemem"
+      "title": "Any extra description for this Item"
     },
-    "_internal_notes": {
+    "internal_notes": {
       "type": "string",
       "title": "Any extra description for this Item reserved for library usage"
     },
-    "circulation_restriction": {
+    "legacy_id": {
       "type": "string",
-      "enum": ["NO_RESTRICTION", "FOR_REFERENCE_ONLY"],
-      "title": "The circulation restriction on this Item"
+      "title": "The previous id of this Item before migration"
+    },
+    "legacy_library_id": {
+      "type": "string",
+      "title": "The previous id of the location of this Item before migration"
     },
     "medium": {
       "type": "string",
       "enum": ["NOT_SPECIFIED", "ONLINE", "PAPER", "CDROM", "DVD", "VHS"],
       "title": "The medium of this Item"
     },
+    "number_of_pages": {
+      "minimum": 1,
+      "title": "Number of pages of the book",
+      "type": "integer"
+    },
+    "physical_description": {
+      "description": ":MARC: ``300`` when no pagination",
```

Comment:
I would rather avoid to add `MARC` references, because it is related more to out environment and migration, unless it is needed, WDYT?